cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -658,6 +662,25 @@ public static KafkaCluster fromCrd(Kafka kafkaAssembly, KafkaVersion.Lookup vers
         return result;
     }
 
+    protected static boolean validateIntConfigProperty(String propertyName, KafkaClusterSpec kafkaClusterSpec) {
```

Comment:
This seems a bit weird now. You validate it, but return boolean and do nothing with it. Shouldn't you throw an exception or something? Eventually, if you agreed with @tombentley to only print warning I can live with it, but why return boolean then? Or did I missed something?

Reply:
If the user configuration is wrong and the Kafka cluster is not going to work, I agree with Jakub that an exception could be raised. It should avoid having the reconcile ends well and starting an unusable cluster , right?

Reply:
It could be a void method but I did not find a better way hoe to test its output via unit test. Raising an exception might be better as it can be expected in the test.

Reply:
`io.strimzi.test.logging.TestLogger` is something I wrote ages ago to be able to test the `ValidationVisitor`, maybe we could use it for testing this too?

Reply:
I was looking for something like this. However, now the exception is used. Is that ok?